docker-compose.yml

```diff
@@ -11,11 +11,13 @@ services:
       SERVICE_NAME: ${SERVICE_NAME}
       DATABASE_USER: ${DATABASE_USER}
       DATABASE_PASSWORD: ${DATABASE_PASSWORD}
+      KEYCLOAK_SERVER_REALM: ${KEYCLOAK_SERVER_REALM}
     hostname: service-api
-    network_mode: "host"
     dns: 142.22.202.100
     ports:
       - "8090:8090"
     build:
       context: .
+      args:
+        KEYCLOAK_SERVER_REALM: ${KEYCLOAK_SERVER_REALM}
```

Comment:
Why are we passing down the realm during build time?
This will "hardcode" one specific realm on the image, making it part of the image itself. For something that will/could change depending on the environment, it is better to use an environment variable or even an external file configuration.

Reply:
Yep, no needed anymore.